test/val/CMakeLists.txt

```diff
@@ -192,3 +192,9 @@ add_spvtools_unittest(TARGET val_non_uniform
         ${VAL_TEST_COMMON_SRCS}
   LIBS ${SPIRV_TOOLS}
 )
+
+add_spvtools_unittest(TARGET val_interfaces
```

Comment:
Please rebase against master.  Insert this new test file in the appriopriate list:   val_ijklmnop

Reply:
Done